helm-charts/0.21.6/values.yaml

```diff
@@ -52,81 +52,14 @@ global:
   vsecm:
     namespace: vsecm-system
   spire:
+    # The trust domain is the root of the SPIFFE ID hierarchy. It is used to
+    # identify the trust domain of a workload. If you use anything other than
+    # the default `vsecm.com`, you must also update the relevant environment
+    # variables that does SPIFFE ID validation.
+    trustDomain: "vsecm.com"
     namespace: spire-system
     logLevel: DEBUG
     serverPort: 8081
 
-
 podAnnotations:
   "helm.sh/hook": pre-install
-
```

Comment:
Removed these as we are not using them right now.

We can add them back when we decide-to and/or start using them.